nextcord/application_command.py

```diff
@@ -2911,7 +2912,7 @@ def slash_command(
     *,
     name_localizations: Optional[Dict[Union[Locale, str], str]] = None,
     description_localizations: Optional[Dict[Union[Locale, str], str]] = None,
-    guild_ids: Optional[Iterable[int]] = None,
+    guild_ids: Optional[Iterable[int]] = MISSING,
```

Comment:
There are other places in the code with a `guild_ids` parameter. While it may not be as easily instantiated, should these also default to `MISSING`?

Example:
![image](https://user-images.githubusercontent.com/20955511/198924266-7ae79626-694b-4b1e-a36d-32c2f29552ad.png)